tests/Makefile

```diff
@@ -1,4 +1,4 @@
-MBEDTLS_TEST_PATH = .
+mbEDTLS_TEST_PATH = .
```

Comment:
Hmm
This doesn't look right to me. Thankfully that part doesn't seem to be in the release candidate.

Reply:
Aw, I must have made a stupid mistake while resolving conflicts, thanks for catching it.

Reply:
Bah, I read that change the wrong way round :(